src/file_utils.cr

```diff
@@ -305,16 +305,21 @@ module FileUtils
   end
 
   # Moves *src_path* to *dest_path*.
+  # If *src_path* and *dest_path* exist on different disk partitions,
```

Comment:
```suggestion
  # If *src_path* and *dest_path* exist on different mounted filesystems,
```

Renaming between different mount points fails, even if the same filesystem is mounted. I know this is copied from the Ruby docs, but they seem to be inaccurate.
See `EXDEV` in [`rename(2)`](https://man7.org/linux/man-pages/man2/rename.2.html).

Reply:
Thanks straight-shoota